.gitignore

```diff
@@ -1,2 +1,3 @@
 coverage.txt
 bin
+.idea/
```

Comment:
this should be on your `~/.gitignore`

Reply:
I don't understand this comment. If you are going to have more people commit, the .gitignore should take into consideration the common editors. If someone uses VSCode, there will be a set of files that should be ignored in that too.